src/test/java/seedu/duke/parsermanager/ParseAddPropertyTest.java

```diff
@@ -0,0 +1,180 @@
+//@@author OVReader
+
+package seedu.duke.parsermanager;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.ClientList;
+import seedu.duke.PairingList;
+import seedu.duke.PropertyList;
+import seedu.duke.Storage;
+import seedu.duke.Ui;
+import seedu.duke.command.Command;
+import seedu.duke.exception.AddressFormatUnitTypeMismatchException;
+import seedu.duke.exception.DukeException;
+import seedu.duke.exception.DuplicatePropertyException;
+import seedu.duke.exception.EmptyAddPropertyDetailException;
+import seedu.duke.exception.IncorrectAddPropertyFlagOrderException;
+import seedu.duke.exception.InvalidPriceFormatException;
+import seedu.duke.exception.InvalidSingaporeAddressException;
+import seedu.duke.exception.InvalidUnitTypeLabelException;
+import seedu.duke.exception.MissingAddPropertyFlagException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ParseAddPropertyTest {
+    // Valid Add Property Test Cases with varying Address Formats
```

Comment:
The name of the constant should be quite intuitive already, thus perhaps there isn't a need for this line of comment anymore?